CONTRIBUTING.md

```diff
@@ -224,6 +224,26 @@ labels.
   have a new feature you'd like to add, you can start a conversation [in our
   development community](https://zulip.com/development-community/#where-do-i-send-my-message)
   explaining the feature idea and the problem that you're hoping to solve.
+- **I think my PR is done, but it hasn't been merged yet. What's going on?**
+  1. **Double-check that you have addressed all the feedback**, including any comments
+     on [Git commit
+     discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-discipline).
+  2. If all the feedback has been addressed, did you leave a comment explaining that
+     you have done so and **requesting a re-review**? If not, it may not be a
+     clear to project maintainers that your PR is ready for another look.
+  3. It is common for PRs to require **multiple rounds of review**. For example,
+     prior to getting code review from project maintainers, you may receive
+     feedback on the UI (without regard for the implementation), and your code
+     may be reviewed by other contributors. This helps us make good use of
+     project maintainers' time, and helps you make progress on the PR by
+     getting more frequent feedback.
+  4. If you think the PR is ready and haven't seen any updates for a couple
+     of weeks, it can be helpful to post a **comment summarizing your
+     understanding of the state of the review process**.
```

Comment:
Maybe add "The ideal such comment will highlight any outstanding questions or decisions, link to relevant chat.zulip.org threads related to outstanding decisions/details, provide updates on screenshots and manual testing if appropriate, and otherwise provide the information one would need to evaluate whether the change is complete." 

One category of issue I see often is folks posting "updated!" without making clear whether they redid the manual testing to verify that the change works as intended, which could be a good example here.